server/prisma/schema.prisma

```diff
@@ -230,6 +231,20 @@ model seasontickettype {
   seasontickets      seasontickets[]
 }
 
+/// price of ticket type for all events in a given season
+model seasontickettypepricedefault {
+  id                 Int                  @id @default(autoincrement())
+  seasonid_fk        Int
+  tickettypeid_fk    Int
+  price              Decimal              @default(0) @db.Money
+  concessionprice    Decimal              @default(0) @db.Money
+  season             seasons              @relation(fields: [seasonid_fk], references: [seasonid], onDelete: Cascade, onUpdate: Cascade)
+  tickettype         tickettype           @relation(fields: [tickettypeid_fk], references: [tickettypeid], onDelete: Cascade, onUpdate: Cascade)
```

Comment:
Why does `Cascade` do for onDelete?

Reply:
If the ticket type being referenced is deleted the season ticket type price can not exist because there is no longer a ticket type to modify the price/concession price for. Similarly, if the season referenced is deleted there is no longer a season to attach the modified prices to.

Reply:
Ok perfect!